### PR TITLE
Updated the readme file.

### PR DESCRIPTION
**Pull request by anchalchaturvedi08**
I have updated the readme file. @Vinay-Khanagavi 
#28 issue no.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,72 +1,61 @@
...
+- **Businesses**: CyberBear offers tailored training programs that educate employees on cybersecurity best practices, bolstering organizational security.
+
+## Setup and Installation
+1. **Clone the Repository**:
+    ```bash
+    git clone https://github.com/yourusername/CyberBear.github.io.git
+    cd CyberBear.github.io
+    ```
+
+## Contribution Guidelines
```
Comment (B1N4RY-P4R45173) [inline README.md:44]: Please add these to a new file `CONTRIBUTING.md`

File: README.md [hunk 2]
```diff
@@ -1,72 +1,61 @@
...
+We welcome contributions to enhance CyberBear! Here’s how you can get started:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Submit a pull request with a clear description of your changes.
+
+### Development Requirements
+- **HTML & CSS**
+- **JavaScript**
+- **Python** (latest stable version)
```
Comment (B1N4RY-P4R45173) [inline README.md:54]: You can remove these lines as, we don't use python for anything as of now

File: README.md [hunk 3]
```diff
@@ -1,72 +1,61 @@
...
+- Any additional packages as noted in `requirements.txt`
+
+## License
+This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for more details.
+
+![CyberBear](https://github.com/Vinay-Khanagavi/CyberFit.github.io/assets/116386393/20f86349-0d7c-4db1-a9bb-7c34d7b3f206)
```
Comment (B1N4RY-P4R45173) [inline README.md:60]: Could You perhaps take more recent screen shots and add them

File: README.md [hunk 4]
```diff
@@ -1,72 +1,61 @@
...
+
+## Use Cases
+- **Individuals**: CyberBear provides a fun way to improve cybersecurity skills through games and quizzes, empowering users to confidently handle online threats.
+- **Educational Institutions**: Interactive modules integrate easily into curricula, enabling educators to provide hands-on cybersecurity education for students.
+- **Businesses**: CyberBear offers tailored training programs that educate employees on cybersecurity best practices, bolstering organizational security.
+
+## Setup and Installation
+1. **Clone the Repository**:
+    ```bash
+    git clone https://github.com/yourusername/CyberBear.github.io.git
```
Comment (B1N4RY-P4R45173) [inline README.md:40]:
```suggestion
    git clone https://github.com/Vinay-Khanagavi/Cyberbear.github.io.git
```
https://github.com/Vinay-Khanagavi/Cyberbear.github.io.git

File: README.md
Comment (B1N4RY-P4R45173) [inline README.md:1]: Also add the lick to our discord server https://discord.gg/ZA7VSKHb9r
Also it would be great if you can add some gifs or something to enhance the UX

**Review by B1N4RY-P4R45173 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (anchalchaturvedi08) [reply README.md:44]: ok

File: README.md (hunk 2 above)
Comment (anchalchaturvedi08) [reply README.md:54]: yeah sure.

File: README.md (hunk 3 above)
Comment (anchalchaturvedi08) [reply README.md:60]: did you mean website images?

File: README.md
Comment (anchalchaturvedi08) [reply README.md:1]: ok I'll add the link and relevant gif too.

File: README.md [hunk 5]
```diff
@@ -34,28 +37,24 @@ CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Pro
 - **Educational Institutions**: Interactive modules integrate easily into curricula, enabling educators to provide hands-on cybersecurity education for students.
 - **Businesses**: CyberBear offers tailored training programs that educate employees on cybersecurity best practices, bolstering organizational security.
 
+![Educational Modules](https://media.giphy.com/media/l3vR3S8rL6rVSHgSY/giphy.gif)
+
+
```
Comment (B1N4RY-P4R45173) [inline README.md:42]:
```suggestion

```

File: README.md [hunk 6]
```diff
@@ -34,28 +37,24 @@ CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Pro
...
-2. Create a new branch for your feature or bug fix.
-3. Submit a pull request with a clear description of your changes.
+[![Join Our Discord](https://img.shields.io/badge/Discord-Join%20Our%20Community-7289DA?style=for-the-badge&logo=discord)](https://discord.gg/ZA7VSKHb9r)
 
-### Development Requirements
-- **HTML & CSS**
-- **JavaScript**
-- **Python** (latest stable version)
-- Any additional packages as noted in `requirements.txt`
+![Community](https://media.giphy.com/media/3o7TKFv9t9O4af7Tzq/giphy.gif)
```
Comment (B1N4RY-P4R45173) [inline README.md:53]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -34,28 +37,24 @@ CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Pro
...
 
 ## License
 This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for more details.
 
-![CyberBear](https://github.com/Vinay-Khanagavi/CyberFit.github.io/assets/116386393/20f86349-0d7c-4db1-a9bb-7c34d7b3f206)
+![CyberBear Banner](https://github.com/Vinay-Khanagavi/CyberFit.github.io/assets/116386393/20f86349-0d7c-4db1-a9bb-7c34d7b3f206)
+
```
Comment (B1N4RY-P4R45173) [inline README.md:59]: Can you change the demo pics to the updated UI. U can get them from
https://vinay-khanagavi.github.io/Cyberbear.github.io/index.html

File: README.md [hunk 8]
```diff
@@ -10,20 +10,23 @@ Welcome to **CyberBear**, a gamified platform designed to educate users on cyber
...
 ## Features
 - **Gamified Learning**: Fun, interactive games to teach cybersecurity concepts.
 - **Subscription Model**: Additional tools and resources for subscribed users.
 - **Quizzes with Feedback**: Test knowledge with quizzes and get instant feedback.
 - **Chatbot Assistant**: 24/7 assistance via a fine-tuned LLM model.
 - **Tools & Resources**: Curated cybersecurity resources for individuals and businesses.
 
+![Quizzes with Feedback](https://media.giphy.com/media/l0HlSNOxJB956qwfK/giphy.gif)
+
```
Comment (B1N4RY-P4R45173) [inline README.md:29]: I don't understand how these gifs are related to out work?

File: README.md [hunk 9]
```diff
@@ -10,20 +10,23 @@ Welcome to **CyberBear**, a gamified platform designed to educate users on cyber
...
-  - [Contribution Guidelines](#contribution-guidelines)
-    - [Development Requirements](#development-requirements)
+  - [Join Our Community](#join-our-community)
   - [License](#license)
 
 ## Project Overview
 CyberBear is part of **HackToFuture-2024**, under the Cyber Security domain (Problem Statement Code: CS-01). Our team aims to promote cybersecurity awareness by turning complex cybersecurity concepts into interactive games, utilizing a subscription model to unlock advanced features, and providing quizzes and real-time assistance with the help of a fine-tuned Large Language Model (LLM).
 
+![Gamified Learning](https://media.giphy.com/media/3o7abB06u9bNzA8lu8/giphy.gif)
+
```
Comment (B1N4RY-P4R45173) [inline README.md:20]: This gif doesn't seem to be related to this project?

**Review by B1N4RY-P4R45173 — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (anchalchaturvedi08) [reply README.md:59]: Ok, I'll change it.

File: README.md (hunk 8 above)
Comment (anchalchaturvedi08) [reply README.md:29]: So, should I remove all the gifs or should I add relevant gifs?

File: README.md (hunk 9 above)
Comment (anchalchaturvedi08) [reply README.md:20]: I'll remove this.

File: README.md (hunk 8 above)
Comment (B1N4RY-P4R45173) [reply README.md:29]: Add the relevant ones

**Review by B1N4RY-P4R45173 — APPROVED** [review]
I think we need to update the demo and add relevant gifs and animation. But we need urgently need this readme file. So I will be merging this. Feel free to open up a new issue for adding the relevant gifs ,animations and transitions.

**Comment by anchalchaturvedi08** [comment]
ok, but I have added the gif and demo video was it not relevant or not good?

**Comment by B1N4RY-P4R45173** [comment]
> ok, but I have added the gif and demo video was it not relevant or not good?

I didn't feel it looked good. And moreover for the demo, a few screenshots would do we don't need a whole video for it.

**Comment by anchalchaturvedi08** [comment]
Ok, So tell me should I work again on it ?
